headless-server-commerce-parent/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>3.3.1</version>
+    <version>3.3.5</version>
```

Comment:
Current spring boot version is 3.4.0. Give it a try.

Reply:
funzt